actionview/lib/action_view/lookup_context.rb

```diff
@@ -54,24 +54,17 @@ module Accessors #:nodoc:
     register_detail(:handlers) { Template::Handlers.extensions }
 
     class DetailsKey #:nodoc:
-      alias :eql? :equal?
+      @store = Concurrent::Map.new
 
-      @details_keys = Concurrent::Map.new
+      # Make hash uniq.
+      alias_method :hash, :object_id
```

Comment:
I don't understand this.

The whole point of this is to provide a key we can use to look up the value in the cache. If it has a new [hash] value each time, the cache is useless. :confused: